chapters/annotations.tex

```diff
@@ -1364,64 +1383,91 @@ \subsubsection{Edit Input}\label{edit-input}
+{\lstinline!defaultConnectionStructurallyInconsistent!} & Relax verification of non-simulation class & \Cref{modelica:defaultConnectionStructurallyInconsistent}\\
+{\lstinline!obsolete!} & Message when using obsolete class & \Cref{modelica:obsolete}\\
+{\lstinline!unassignedMessage!} & Hint for unmatched variable & \Cref{modelica:unassignedMessage}\\
+{\lstinline!Dialog!} & Setup for component modifications & \Cref{modelica:Dialog}\\
+\hline
+\end{tabular}
+\end{center}
 
-\begin{lstlisting}[language=modelica]
+\begin{annotationdefinition}[preferredView]
+\begin{synopsis}\begin{lstlisting}
 /*literal*/ constant String preferredView;
-\end{lstlisting}
-
+\end{lstlisting}\end{synopsis}
+\begin{semantics}
 The \fmtannotationindex{preferredView} annotation defines the default view when selecting the class.
-The $\mathit{view}$ is a \lstinline!String! literal where \lstinline!"info"! means class documentation (``information''), \lstinline!"diagram"! means diagram view, \lstinline!"icon"! means icon view, and \lstinline!"text"! means Modelica source code (``text'').
+The value \lstinline!"info"! means class documentation (``information''), \lstinline!"diagram"! means diagram view, \lstinline!"icon"! means icon view, and \lstinline!"text"! means Modelica source code (``text'').
 If not specified the default view is tool-specific.
+\end{semantics}
+\end{annotationdefinition}
 
-\begin{lstlisting}
+\begin{annotationdefinition}[DocumentationClass]
+\begin{synopsis}\begin{lstlisting}
 /*literal*/ constant Boolean DocumentationClass;
-\end{lstlisting}%
-\annotationindex{DocumentationClass}
-
+\end{lstlisting}\end{synopsis}
+\begin{semantics}
 Only allowed as class annotation on any kind of class and only having effect when \lstinline!true!, meaning that this class and all classes within it are treated as having the annotation \lstinline!preferredView = "info"!.
 If the annotation \lstinline!preferredView! is explicitly set for a class, it has precedence over a \lstinline!DocumentationClass! annotation.
 
 \begin{nonnormative}
 A tool may display such classes in special ways.
 For example, the description texts of the classes might be displayed instead of the class names, and if no icon is defined, a special information default icon may be displayed in the package browser.
 \end{nonnormative}
+\end{semantics}
+\end{annotationdefinition}
 
-\begin{lstlisting}[language=modelica]
+\begin{annotationdefinition}[defaultComponentName]
+\begin{synopsis}\begin{lstlisting}
 /*literal*/ constant String defaultComponentName;
-\end{lstlisting}%
-\annotationindex{defaultComponentName}
-
-When creating a component of the given class, and the annotation is specified it gives the recommended component name.
+\end{lstlisting}\end{synopsis}
+\begin{semantics}
+\lstinline!defaultComponentName! is a class annotation giving the recommended component name to use when creating a component of the class.
```

Comment:
```suggestion
The \lstinline!defaultComponentName! is a class annotation giving the recommended component name to use when creating a component of the class.
```
Avoid starting a sentence with a lower-case letter.

Reply:
There are numerous examples of starting sentences like this with a lower case initial in inline code, so I don't think we should start using another style here.

Reply:
Well, 109 lines starting with `The \lstinline`, 28 `A \lstinline` and 4 `An \lstinline`
And there are only 76 lines starting with `\lstinline` followed by a lower-case letter - many of them not being real sentences, but just items in a list.

Reply:
Sure, but I hope that of the 109, a majority are such that it comes natural to formulate using the definite form.  Here, it just looks lika a contrived artifact of applying a rule which was not made with technical texts like ours in mind.

Reply:
On the contrary, it is applied in many technical texts like ours, because it reduces the risk of errors and misinterpretations.

Obviously the sentence can be reformulated to be natural and more direct:
```suggestion
The class annotation \lstinline!defaultComponentName! gives the recommended component name to use when creating a component of the class.
```

In addition to the 109 lines starting with `The \lstinline`, there are 62 lines starting with `The <word> \lstinline`, and 54 lines with two words, 43 lines with three words, and 31 with four words (and some lines starting with A/An followed by some words as well.)